core/src/test/java/com/google/instrumentation/trace/SamplersTest.java

```diff
@@ -25,9 +26,9 @@
 public class SamplersTest {
   @Test
   public void alwaysSampleSampler_AlwaysReturnTrue() {
-    TraceId traceId = new TraceId(2, 3);
-    SpanId parentSpanId = new SpanId(17);
-    SpanId spanId = new SpanId(42);
+    TraceId traceId = TraceId.generateRandomId(new SecureRandom());
+    SpanId parentSpanId = SpanId.generateRandomId(new SecureRandom());
+    SpanId spanId = SpanId.generateRandomId(new SecureRandom());
```

Comment:
If I understand the documentation correctly, `SecureRandom` can produce non-deterministic results even if the seed is set, so I would use `java.util.Random`.  I would also create a new `Random` at the start of every test case, either as a local variable or an instance variable that can be initialized in a `@Before` method, so that test order doesn't affect the results.